lerna.json

```diff
@@ -5,5 +5,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.51.1"
+  "version": "2.0.0"
```

Comment:
Let's revert this line.

The CI system sets this based off the tag that is created through the GH Releases page.

Reply:
https://github.com/yaorg/node-measured/blob/master/scripts/publish.sh#L94

Reply:
Thanks. Reverted the version back to 1.51.1